README.md

```diff
@@ -0,0 +1,23 @@
+# Battlesnake Admin
+This project is for the admin dashboard of the 2018 [battlesnake.io](https://battlesnake.io) coding event.
+It's gameserver lives at http://github.com/battle-snake/battle_snake.
+
+## getting started
+pull the repo, then run `./scripts/setup`
+
+### Create some files
+
+#### `./scripts/env-local.yml` with this content:
+```yaml
+
+```
+
+#### `./scripts/.env` with this content:
```

Comment:
we aren't using a .env file for this project, doesn't work well for the serverless framework